README.md

```diff
@@ -435,13 +506,69 @@ The Zenon Wallet API is a production-ready solution that provides a secure inter
    - Track address balances
    - Monitor address activity
 
-### Getting Started
+#### Wallet API System Requirements
+- **CPU**: 2+ cores
+- **RAM**: 4GB minimum
+- **Storage**: 20GB SSD minimum
+- **Network:** 100Mbps minimum, stable connection
+- **Operating System**: Ubuntu 22.04 LTS or newer
+
+## Node Operations
+The CEX will need to deploy a local node to support their wallet infrastructure. The Hypercore One community developers maintain a `go-zenon` node [deployment script](https://github.com/hypercore-one/deployment). Follow the instructions there to deploy a node.  Also, Zenon Network community developers maintain a [Zenon Node Controller](https://github.com/zenon-network/znn_controller_dart) to deploy and maintain `go-zenon`.
```

Comment:
whats the difference between the deployment script and the controller. Is the controller still necessary?

Reply:
the controller script was developed by the founding developers.  It could be confusing to include it.  The deploy script does more.  I'll remove reference to the dart controller app.